.github/workflows/wait_for_required_workflows.yml

```diff
@@ -42,12 +42,13 @@ jobs:
             let actions = ["cli",
                            "scaffold",
                            "plugins/source/alicloud",
+                           "plugins/source/awspricing",
```

Comment:
There's no need to sort these

Reply:
If they are not sorted, then merge conflicts can occur (which is what happened to me)

Reply:
Agreed, but my feeling is that this will leave it better for the next person that tries to make a change here

Reply:
Happy to finally see these sorted 👍 💯